test/tests/integration/phold.c

```diff
@@ -22,6 +24,10 @@
 
 #define EVENT 1
 
+struct phold_state {
+	struct drand48_data rng_state;
```

Comment:
I disagree on this. I was purposely avoiding the test RNG in this test, to make the model independent of the testing framework. This could serve as a baseline for model developers.

That said, the version of the RNG that I used is a GNU extension, and was picked in error. I noticed that but did not have time to fix it yet. It will be addressed in another commit.